src/appmain.ts

```diff
@@ -741,18 +713,15 @@ export class ReplayWebApp extends LitElement {
                       IS_APP
                         ? html`
                             <p>
-                              ReplayWeb.page App is a standalone app for Mac,
-                              Windows and Linux that loads web archive files
+                              ReplayWeb.page is a standalone app for Mac,
```

Comment:
This is less clear, 'archived items' is what we call things in our app, but what we are dealing with is allowing users to load web archive files, which we then refer to as archived items. In general, archived items could be anything, not related to web archives